packages/atomizer/src/helpers.js

```diff
@@ -283,9 +275,7 @@ module.exports = [
             },
             'a[class*=LineClamp]': {
                 'display': 'inline-block',
-                'display ': '-webkit-box',
-                '*display': 'inline',
-                'zoom': 1
+                'display ': '-webkit-box'
```

Comment:
@thierryk @carriemorrison `-webkit-box` is a non standard thing, should we remove all references now?

Reply:
> `-webkit-box` is a non standard thing...

Is it?  /s

[6.1.1 Legacy Compatibility](https://www.w3.org/TR/css-overflow-3/#webkit-line-clamp)

> UAs that support [line-clamp](https://www.w3.org/TR/css-overflow-3/#propdef-line-clamp) must also support the [-webkit-line-clamp](https://www.w3.org/TR/css-overflow-3/#propdef--webkit-line-clamp)

They made a mess when they introduced vendor prefixes. i.e., MS Edge supports `line-clamp` with `-webkit-` prefix, but not with `-ms`...

I'd leave things the way they are though as this original `-webkit-` way to do line-clamp was really a hack. 
And also because for this to work, we are using `-webkit-box-orient` which is another prefixed property.

Better would be to revisit the styling/declarations in that "helper" class to use the new stuff now that we have [`line-clamp`](https://www.w3.org/TR/css-overflow-3/#propdef-line-clamp) and other goodies...